### PR TITLE
fix: correct responsive padding in some consumers of popups

### PR DESCRIPTION
**Pull request by gmkv**
Affected components:
 - NotificationListItem
 - NotificationListGroupItem
 - SideNavigation
 - Wizard
 - Consumers of ResponsivePopover - footer padding is corrected

Added left/center aligned titles to the Popover and Dialogs for the SAP Quartz (centered in Belize) themes when the property `headerText` is provided.
Improved some samples.

Part of #4870

### CODE REVIEW

**Review by ivoplashkov — COMMENTED** [review]
Wizard looks good to me.

**Review by dimovpetar — COMMENTED** [review]
LGTM, just clarify in the padding of the popover in notifications should differ based on the device (desktop and phone/tablet), or differ based on the density mode (cozy and compact)

**Comment by gmkv** [comment]
> LGTM, just clarify in the padding of the popover in notifications should differ based on the device (desktop and phone/tablet), or differ based on the density mode (cozy and compact)

The design team confirmed that the padding for the notifications should be based on the content density, not on the device breakpoint size.

File: packages/fiori/src/NotificationListItemBase.js [hunk 1]
```diff
@@ -12,10 +12,10 @@ import "@ui5/webcomponents-icons/dist/message-warning.js";
 import "@ui5/webcomponents-icons/dist/overflow.js";
 
 // Templates
-import NotifactionOverflowActionsPopoverTemplate from "./generated/templates/NotifactionOverflowActionsPopoverTemplate.lit.js";
```
Comment (TeodorTaushanov) [inline packages/fiori/src/NotificationListItemBase.js:15]: Change "Notifaction" to "Notification" everywhere.

Comment (gmkv) [reply packages/fiori/src/NotificationListItemBase.js:15]: Already done.

**Review by TeodorTaushanov — APPROVED** [review]

**Pull request merged**